project-management/src/test/groovy/life/qbic/projectmanagement/application/policy/SampleDeletedPolicySpec.groovy

```diff
@@ -17,13 +19,15 @@ import spock.lang.Specification
 
 class SampleDeletedPolicySpec extends Specification {
 
-    def "Given a sample deletion event, the directive to remove the sample from a batch is executed"() {
+    def "Given a sample deletion event, the respective directive is executed"() {
```

Comment:
why the name change?
Maybe it would make sense to test them separately so they can fail separately.

Reply:
good point, I changed the name and added another test class for project changed events